api/index.html

```diff
@@ -210,8 +210,8 @@ <h2 id="application-adapter">Application Adapter</h2>
 
 <p>We're using an Ember Data built-in adapter called the JSONAPIAdapter. Building a custom adapter isn't too hard, but we don't need to because we're going to use the default.</p>
 <p>Finally, to point our Ember app at the API we've set up, let's go back to our server, hit <code>CTRL-C</code> to stop it, and restart <code>ember serve</code> using the proxy option to point Ember to the API we want to access:</p>
-<pre><code class="console">$ ember serve --proxy https://ember-101-api.herokuapp.com
-Proxying to https://ember-101-api.herokuapp.com
+<pre><code class="console">$ ember serve --proxy https://sandiego-ember-cli-101.herokuapp.com
```

Comment:
Nope! New ember endpoing, the old one doesn't work anymore. I updated with a new Rails 5 & JSON API backend.